### PR TITLE
hnsAcls is the default and only option

### PR DESCRIPTION
**Pull request by winkingturtle-vmw**
Compiling this binary with Firewall had been deprecated long ago and couldn't be compiled without hnsAcls tag otherwise. https://github.com/cloudfoundry/winc/commit/bedd8b61ad954a08beb6b3a0225306893f731892

### CODE REVIEW

File: cmd/winc-network/main_acl.go [hunk 1]
```diff
@@ -11,10 +11,3 @@ import (
 	"code.cloudfoundry.org/winc/network/port_allocator"
```
Comment (mariash) [inline cmd/winc-network/main_acl.go]: Is this file needed?

Comment (winkingturtle-vmw) [reply cmd/winc-network/main_acl.go]: No, I've now deleted this file
